src/target/riscv/riscv-013.c

```diff
@@ -199,6 +205,13 @@ typedef struct {
 
 	/* This target was selected using hasel. */
 	bool selected;
+
+	/* When false, we need to set dcsr.ebreak*, halting the target if that's
+	 * necessary. */
+	bool dcsr_ebreak_set;
```

Comment:
Could we please rename this to `is_dcsr_ebreak_set`? To make it very clear that `true` means the work is done. (Not that `true` means a pending action that is yet to be done.)

```suggestion
	bool is_dcsr_ebreak_set;
```

or 

```suggestion
	bool dcsr_ebreak_is_set;
```

Reply:
Yes. That's more clear. Thanks.

Reply:
@timsifive Can you please check if you pushed your changes? I can see here just the older revision.